stable/traefik/README.md

```diff
@@ -88,7 +88,8 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `image`                         | Traefik image name                                                   | `traefik`                                 |
 | `imageTag`                      | The version of the official Traefik image to use                     | `1.3.1`                                  |
 | `serviceType`                   | A valid Kubernetes service type                                      | `LoadBalancer`                            |
-| `loadBalancerIP`                | An available static IP you have reserved on your cloud platform      | None                                      |
+| `loadBalancer.IP`               | An available static IP you have reserved on your cloud platform      | None                                      |
```

Comment:
@unguiculus That was the original PR. @dhilipkumars asked me to change it to add nesting, so that went in 68324a7f835e1b0beec1dbe4e5bb5a2ca98c595d. 
I can revert that commit and leave it as it was originally intended.

Would that be OK with everyone?

Thank you.